plugins/outputs/sql/sql_test.go

```diff
@@ -288,7 +337,6 @@ func TestPostgresIntegration(t *testing.T) {
 	require.NoError(t, p.Connect())
 	defer p.Close()
 	require.NoError(t, p.Write(testMetrics))
-	require.NoError(t, p.Close())
```

Comment:
Why do you remove this? This line makes sure all metrics are written...

Reply:
It closes the connection and i want to re-use the existing connection, instead of setting up a new one.

The tests seem to pass fine. Are you sure Close is required to flush the metrics? Is this a pgx quirk?

Reply:
Yes this is on purpose! If `Close` finishes we know all commands are flushed to the server and as I said below, please create an own test-case for your stuff!